src/pyfmi/fmi.pyx

```diff
@@ -728,15 +738,24 @@ cdef class ModelBase:
         if self._additional_logger:
             self._additional_logger(module, log_level, message)
 
-        full_msg = "FMIL: module = %s, log level = %d: %s\n"%(module, log_level, message)
+        if self._max_log_size_msg_sent:
+            return
+
+        msg = "FMIL: module = %s, log level = %d: %s\n"%(module, log_level, message)
+
+        if self._current_log_size + len(msg) > self._max_log_size:
```

Comment:
```suggestion
        if self._current_log_size + len(msg) >= self._max_log_size:
```

Reply:
I'd actually keep this as is.

My interpretation around most language around log size limits in doc strings is that the limit is inclusive, i.e., we only stop logging after the limit is passed. This would change it to a non-inclusive limit + it would also require more adjustments in other place.

Reply:
No problem, let's stick with your proposal.